target/murabi/WEB-INF/web.xml

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE web-app PUBLIC "-//Sun Microsystems, Inc.//DTD Web Application 2.3//EN" "http://java.sun.com/dtd/web-app_2_3.dtd">
```

Comment:
[major] Don't commit anything from the target dir since it contains only items that are built by maven.